README.md

```diff
@@ -0,0 +1,17 @@
+# otm-website-tests
+
+An automated test suite for the OpenTreeMap website.
+
+## Usage
+
+To execute the test suite, first install [Leiningen](http://leiningen.org/), then execute the following command:
```

Comment:
The leiningen install instructions do not mention that you need a JVM. For those totally new to Clojure, it would be a good idea to list that dependency here.